Limma: small fixes

mblue9 opened the pull request:
* Fixes difference of differences contrasts (broke when I changed to code to allow groups to start with numbers)
* Fixes links to tsvs and rdata showing up under Plots section of report

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

bgruening commented on tools/limma_voom/limma_voom.R:
```diff
@@ -353,16 +361,29 @@ if (is.null(opt$contrastInput)) {
-    i <- lapply(i, paste, collapse="-")
-    cons <- append(cons, unlist(i))
+
+    # if the contrast is a difference of differences e.g. (A-B)-(X-Y)
```
You could also just forbid to enter numbers on the user-interface at the beginning of a contrast name.

mblue9 commented:
@bgruening I think I deleted your comment? I changed to allow groups to start with numbers [here](https://github.com/galaxyproject/tools-iuc/pull/2302) to be consistent with the deseq2 tool
